kernel/comps/framebuffer/src/console.rs

```diff
@@ -62,6 +68,19 @@ impl AnyConsoleDevice for FramebufferConsole {
     fn set_font(&self, font: BitmapFont) -> Result<(), ConsoleSetFontError> {
         self.inner.lock().0.set_font(font)
     }
+
+    fn set_mode(&self, mode: ConsoleMode) -> bool {
+        let mut inner = self.inner.lock();
+        let (state, _) = &mut *inner;
+        state.mode = mode;
```

Comment:
To be consistent with the existing `set_font` method, this should be `self.inner.lock().0.mode = mode`.